airbyte-webapp/src/views/Connector/ServiceForm/ServiceForm.tsx

```diff
@@ -247,44 +201,12 @@ const ServiceForm: React.FC<ServiceFormProps> = (props) => {
         ),
       },
       serviceType: {
-        component: ({ path }: FormBaseItem, componentProps: FormComponentOverrideProps) => {
-          return (
-            <>
-              <ConnectorServiceTypeControl
-                propertyPath={path}
-                formType={formType}
-                onChangeServiceType={onServiceSelect}
-                availableServices={availableServices}
-                isEditMode={isEditMode}
-                onOpenRequestConnectorModal={(name) => {
-                  setInitialRequestName(name);
-                  toggleOpenRequestModal();
-                }}
-                {...componentProps}
-              />
-              {!isEditMode && formType === "destination" && !selectedConnectorDefinitionSpecification && (
-                <FrequentlyUsedDestinations
-                  propertyPath={path}
-                  destinations={frequentlyUsedDestinations}
-                  onDestinationSelect={onServiceSelect}
-                  isLoading={isLoading}
-                />
-              )}
-            </>
-          );
-        },
+        /* since we use <ConnectorServiceTypeControl/> outside formik form
+           we need to keep the serviceType field in formik, but hide it */
+        component: () => null,
```

Comment:
Do we need to define a `null` component?

Reply:
Answer below